src/StatusQ/Components/StatusToastMessage.qml

```diff
@@ -0,0 +1,166 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+import QtGraphicalEffects 1.13
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+
+Control {
+    id: root
+    width: 343
+    height: !!secondaryText ? 68 : 48
+    anchors.right: parent.right
+
+    property bool open: false
+    property string primaryText: ""
+    property string secondaryText: ""
+    property bool loading: false
+    property string iconName: ""
```

Comment:
I think it'd be nicer if we used `StatusIconSettings` here instead and then give it some decent defaults. This way, we stay consistent with other components that render icons.

Reply:
I thought about this, but since is nothing specified yet, I don't know whether it will be only the icon name needed, more or nothing (maybe we have fixed icons). I'd avoid adding complexity if not needed. nevertheless this is something we can always add later when more details are available.

Reply:
I understand your reasoning. Just be aware that, if we do need to change it afterwards, it will introduce a breaking change (which I'd avoid as much as possible) unless we keep the `iconName` property around.

I'll let this be your call.